src/ai/defaultai.cc

```diff
@@ -3735,8 +3672,8 @@ bool DefaultAI::construct_building(const Time& gametime) {
 
 // Re-calculating warehouse to flag distances
 void DefaultAI::check_flag_distances(const Time& gametime) {
+	uint16_t checked_flags = 0;
```

Comment:
it seems this variable is just incremented but never used anywhere. so we could delete it

Reply:
yes, I was using it for debug. removed